ethcore/evm/src/interpreter/gasometer.rs

```diff
@@ -392,7 +392,7 @@ fn calculate_eip1283_sstore_gas<Gas: evm::CostType>(schedule: &Schedule, origina
 				}
 			} else {
 				// 2.2. If original value does not equal current value (this storage slot is dirty), 200 gas is deducted. Apply both of the following clauses.
```

Comment:
This comment is now outdated, can be 200 or 800 (if eip2200_advance_transition is activated)